src/modules/uavcan/uavcan_servers.cpp

```diff
@@ -526,6 +531,18 @@ pthread_addr_t UavcanServers::run(pthread_addr_t)
 					}
 				}
 			}
+
+			// Acknowledge the received command
+			struct vehicle_command_ack_s ack = {};
+			ack.command = cmd.command;
+			ack.result = cmd_ack_result;
+
+			if (_command_ack_pub == nullptr) {
+				_command_ack_pub = orb_advertise_queue(ORB_ID(vehicle_command_ack), &ack, vehicle_command_ack_s::ORB_QUEUE_LENGTH);
```

Comment:
Do we really need to hold the handle here or shouldn't we instead publish and immediately `orb_unadvertise()`? I think that's more appropriate here.